VideoWeb/VideoWeb.EventHub/Handlers/TransferEventHandler.cs

```diff
@@ -25,32 +19,32 @@ public class TransferEventHandler(
 
         protected override async Task PublishStatusAsync(CallbackEvent callbackEvent)
         {
-            var participantStatus = DeriveParticipantStatusForTransferEvent(callbackEvent);
+            var participantStatusTuple = DeriveParticipantStatusForTransferEvent(callbackEvent);
```

Comment:
could we keep this as `participantStatus`?

Reply:
I renamed it to tuple so it's clear what we're getting back. Ideally I want to remove the duplicate models when they're both the same